src/main/java/Duke.java

```diff
@@ -1,10 +1,186 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+    //taskList variable for level-2 onwards
+    private static Task[] taskList = new Task[100];
+    //task count
+    private static int taskCount = 0;
+
+
+    //echo function for level-1
+    public static void echo(String message) {
+        System.out.println(message);
+    }
+
+
+    //add function for level-2
+    public static void add(String task) {
+        Task newTask = new Task(task);
+        taskList[taskCount] = newTask;
+        taskCount++;
+        System.out.println("added: " + task);
+    }
+
+
+    //list function for level-2
+    public static void list() {
```

Comment:
I like how you extracted single functions of the chatbot like listing all the tasks into their own method in the overall class! It helps greatly with modularity.